plugins/out_s3/s3.c

```diff
@@ -1447,6 +1526,18 @@ static struct flb_config_map config_map[] = {
      0, FLB_FALSE, 0,
     "Predefined Canned ACL policy for S3 objects."
     },
+    {
+     FLB_CONFIG_MAP_STR, "compression", NULL,
+     0, FLB_FALSE, 0,
+    "Compression type for S3 objects. 'gzip' is the only support value for now. "
+    "This will be set as the Content-Encoding HTTP header."
+    },
+    {
+     FLB_CONFIG_MAP_STR, "content_type", NULL,
+     0, FLB_FALSE, 0,
+    "A standard MIME type describing the format of the contents. This will be set"
```

Comment:
A standard MIME type for the S3 object; this will be set as the Content-Type HTTP header